frontends/mit-open/src/pages/HomePage/BrowseTopicsSection.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react"
+import { Container, styled, theme, Button, Typography } from "ol-components"
+import { Link } from "react-router-dom"
+import { useTopicsList } from "api/hooks/topics"
+import {
+  RiPaletteLine,
+  RiSeedlingLine,
+  RiBriefcaseLine,
+  RiMacbookLine,
+  RiBarChartBoxLine,
+  RiUserSearchLine,
+  RiArrowRightSLine,
+} from "@remixicon/react"
+
+const ICONS = [
+  RiPaletteLine,
+  RiSeedlingLine,
+  RiBriefcaseLine,
+  RiMacbookLine,
+  RiBarChartBoxLine,
+  RiUserSearchLine,
+  RiArrowRightSLine,
+]
+
+const Section = styled.section`
+  background:
+    linear-gradient(270deg, rgba(243 244 248 / 0%) 0%, rgb(243 244 248) 100%),
+    url("/static/images/open-bg-texture-wgradient.jpg") lightgray 50% / cover
+      no-repeat;
+  background-position: center right;
+  padding: 80px 0;
+  ${({ theme }) => theme.breakpoints.down("md")} {
+    padding: 40px 0;
+  }
+`
+
+const Title = styled(Typography)`
+  text-align: center;
+`
+
+const Topics = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 16px 24px;
+  margin: 40px 0;
+  ${({ theme }) => theme.breakpoints.down("md")} {
+    gap: 5px;
+    margin: 24px 0;
+  }
+`
+
+const TopicBox = styled(Link)`
+  flex: 0 1 calc(100% * (1 / 3) - 16px);
```

Comment:
_Not a requested change_, but btw, also possible to do `<Grid component={Link} to="path/to/link/"> ... </Grid>` if you want to use the grid component. 

It's documented here: https://mui.com/material-ui/guides/composition/#component-prop ... not on each component page b/c it's common to all mui components.